### ISSUE TITLE
--debug-brk broken as of 2.2.0

### ISSUE THREAD

**Issue by wenzowski** [mochajs/mocha#1587]
on node 0.10.36

```stderr
./node_modules/.bin/mocha -C --debug-brk

events.js:72
        throw er; // Unhandled 'error' event
              ^
Error: spawn ENOENT
    at errnoException (child_process.js:1011:11)
    at Process.ChildProcess._handle.onexit (child_process.js:802:34)
make: [test-debug] Error 8 (ignored)
```

**Comment by wenzowski** [comment]
but working fine with mocha@2.1.0

**Comment by travisjeffery** [comment]
was fixed in 2.2.1! (or should be, try it and let me know). thanks!

**Comment by wenzowski** [comment]
yerp thanks @travisjeffery
